tools/hardhat/README.md

```diff
@@ -0,0 +1,131 @@
+# Precompile Hardhat Tasks
+
+Node.js v20 or greater is recommended
+
+Before getting started make sure you have changed to the hardhat directory and have installed the required
+dependencies:
+
+```bash
+cd tools/hardhat
+npm install
```

Comment:
Just idea:
It'd be nice to include Dockerfile for this setup. Then, users won't need to have Node.js locally.

Reply:
This is a temporary solution. The ultimate one will be a CLI command exposed by the `mezod` binary.

Reply:
Yes, this is only a temp working solution for first validators. Then we'll have a CLI command added. Adding Dockerfile is nice, but delays this toolbox and might not be necessary at the end.